lib/moment-range.js

```diff
@@ -37,6 +37,10 @@ export class DateRange {
 
     this.start = s || s === 0 ? moment(s) : moment(-8640000000000000);
     this.end = e || e === 0 ? moment(e) : moment(8640000000000000);
+
+    if (this.end.isBefore(this.start)) {
+      [this.start, this.end] = [this.end, this.start];
+    }
```

Comment:
This doesn't seem safe to me (e.g. developer calling this makes a typo).

Double-check: have we already considered throwing an error instead?

Reply:
Thanks for discussing 😃 

IMO the library should provide the core time functionality, and the user can easily change their application logic if they're expecting a reversed time range. We classify it as a period between two points in time, rather than a directional period.

Throwing an error isn't ideal because _most_ of the time, the functions on a range behave identically regardless of the start or end. I would think that it's far more inconvenient to require application start/end checks or error handling when they likely aren't requiring reverse time ranges? But I see the issue (if it's not documented clearly) of users not understanding what's happening.

@gf3 do you think it's better to throw error or is it unnecessary?